lib/utils/replace.go

```diff
@@ -128,6 +128,22 @@ func SliceMatchesRegex(input string, expressions []string) (bool, error) {
 	return false, nil
 }
 
+func RegexMatchesSlice(inputs []string, expression string) (bool, error) {
```

Comment:
Add godoc and a few tests?

Maybe call it `RegexMatchesSliceAny` or `RegexMatchesAny`?